docs/getting-started/first-app.md

```diff
@@ -42,7 +42,7 @@ For the purpose of this tutorial, we'll now add another entity called “Project
 
 You now have a new “Project” entity. Notice that the added entity comes with auto-generated values like “Plural Display Name” and some default fields – ID, Created At, and Updated At.
 
-## Add Entity Fields
+## Add Entity Fields {:#add_entity_fields}
```

Comment:
```suggestion
## Add Entity Fields
```

Reply:
@udanna, 
(from here https://stackoverflow.com/questions/6695439/how-to-link-to-a-named-anchor-in-multimarkdown)
If you have headers in the markdown files, you can directly link them in the file.
Markdown Header -
## The Header
this will generate an implicit id #the-header (replace internal spaces with hyphens and make lowercase).
To navigate to this id, you can create the link like this:
[Link to Header](#the-header)